docs/guides/concepts.md

```diff
@@ -1,13 +1,16 @@
 ---
+bodyclass: docs
+headline: gRPC Concepts
 layout: docs
+sidenav: doc-side-guides-nav.html
 title: gRPC Concepts
+type: markdown
 ---
+<p class="lead">This document introduces some key gRPC concepts with an overview of gRPC's architecture and RPC life cycle.</p> 
 
-<h2 class="page-header">gRPC Concepts</h2>
+It assumes that you've read the [Overview]({{site.baseurl}}/docs). For language-specific details, see the Quick Start, tutorial, and reference documentation for your chosen language(s), where available (complete reference docs are coming soon).
```

Comment:
I think it's OK "as is" - as they're all language-specific the only place we could send them would be to the Quickstarts, Tutorials, and Reference landing pages, where they'd need to find their language anyway.